src/stream/video_stream_rtsp.rs

```diff
@@ -0,0 +1,84 @@
+use super::gst::pipeline_runner::Pipeline;
```

Comment:
This commit should not exist, please do the necessary squashing, avoid adding intermediary development commits in master branch.